src/test/euler/staking/StakeableEulerERC4626.t.sol

```diff
@@ -190,6 +191,22 @@ contract StakeableEulerERC4626Test is Test {
         assertEq(address(vault.stakingRewards()), address(0));
     }
 
+    function testTotalAssets() public {
+        uint256 deposited = 10000;
+        _deposit(alice, deposited);
+
+        // Check without staking contract
+        assertEq(vault.totalAssets(), deposited);
+
+        uint256 staked = 1000;
+        _setStakingContract();
+        _stake(staked);
+
+        // Should still be the same as before
+        assertEq(vault.totalAssets(), deposited);
+    }
+
```

Comment:
Would be cool to add a last deposit at the end? to do converted(staked + deposit).

Reply:
Done!